string id
furniture/Multiplanar_Model model
furniture/Pose_2D_With_Z pose
